docs/source/miscellaneous/dropwizard-metrics.rst

```diff
@@ -0,0 +1,61 @@
+.. _dropwizard-metrics:
+
+==================
+Dropwizard Metrics
+==================
+
+AtlasDB makes use of the Dropwizard `Metrics library <http://metrics.dropwizard.io/>`__ to
+expose a global ``MetricRegistry`` called ``AtlasDbRegistry``. Users of AtlasDB should use ``AtlasDbMetrics.setMetricsRegistry``
+to make use of our ``MetricRegistry``.
```

Comment:
I think we want to say that AtlasDB users should use `AtlasDbMetrics.setMetricRegistry` to inject their own MetricRegistry for their application. If they do not inject their own prior to initializing the AtlasDB transaction manager, the metrics are registered to a shared MetricRegistry, which is often not what folks really want.